song_translator/track/migrations/0005_delete_comment.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 3.2 on 2021-05-13 14:48
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('track', '0004_rename_translate_translation'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
```

Comment:
that's fine for now, but in real world strive for least possible number of migrations per PR. I mean you create and delete Comment model in one PR